torch/csrc/utils/python_arg_parser.h

```diff
@@ -302,10 +302,12 @@ struct PythonArgs {
   inline bool isNone(int i);
   inline std::optional<c10::DispatchKeySet> toDispatchKeySetOptional(int i);
 
- private:
```

Comment:
These are not part of the functions that used to be public but moved to private 3 months ago? Why are you adding them here?

Reply:
Do you now who moved them to private?

Reply:
The "moved to private" I mention here is the PR from 3 months ago hiding the symbols. Not moving these to be on this class "private:" tag.

Reply:
I am not sure what do you mean by `public`. Do you mean `public C++ member function` or `symbols available in DLL`? Our custom backend links to these functions when building with PyTorch. Those functions are accessible in PyTorch DLL ~ 1 year ago, so they were exposed in DLL before.

Reply:
@albanD, @malfet, gentle ping. Please let me know if I missed anything.

Reply:
Hmm, just curious, would `TORCH_PYTHON_API` the entire class will work? (That would transitively apply to all methods?)

Reply:
@malfet, I can try but that's not the conclusion made by other meta folks in https://github.com/pytorch/pytorch/issues/148208. Just tell me the change you want.

Reply:
Tagging `TORCH_PYTHON_API` at structure level works for my case.

Reply:
Let's land it this way then